### ISSUE TITLE
Fix lms & cms container environments to run tests without modification

### ISSUE THREAD

**Issue by kdmccormick** [openedx/wg-devops#23]
## Context

Here are the instructions on how to run tests in Tutor: https://docs.tutor.overhang.io/dev.html#running-edx-platform-unit-tests

Notice that:
1. the `lms` container must be used, even for `cms` tests
2. one needs to fiddle with a few different environment variables in order to run `pytest` on each root folder (lms, cms, common, openedx, xmodule).

Both of these limitations are awkward and potential stumbling blocks for developers.

### Note on lms vs cms tests

edx-platform runs in one of two modes: LMS (the learning/instruction interface & APIs) and CMS (Studio and authoring APIs). Some code is specific to LMS or CMS, and some code is shared between the two. Both LMS and CMS have their own Django settings files.

In the edx-platform source tree...

* The tests under ./lms/ are LMS-specific and should only be run with LMS settings.
* The tests under ./cms/ are CMS-specific and should only be run with CMS settings.
* The tests under ./openedx/, ./common/, and ./xmodule/ are "shared". That, the code could be used by both LMS and CMS, so we want to be able to run unit tests *both* using LMS settings and CMS settings, and the tests should pass under both contexts.
  * Exceptions: Some unit tests in these directories are LMS-specific or CMS-specific. Those tests are marked with decorators `@skip_unless_lms` and `@skip_unless_cms`, respectively.

edx-platform's PR checks ensure that the above is always true on the master branch.

## Acceptance

Tweak edx-platform's test setup and/or Tutor's cms+lms dev container environments such that, without having to change any environment variables:

* One can run all LMS-specific tests *and* shared tests in the `lms` container using LMS Django settings.
* One can run all CMS-specific tests *and* shared tests in the `cms` container using CMS Django settings.

In other words, these should just work:

```python
tutor dev run lms pytest lms/ xmodule/ common/ openedx/
tutor dev run cms pytest cms/ xmodule/ common/ openedx/
```
